core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -1240,9 +1244,11 @@ setting.about.shortcuts.sendFilter=Set Filter (privileged activity)
 setting.about.shortcuts.sendPrivateNotification=Send private notification to peer (privileged activity)
 setting.about.shortcuts.sendPrivateNotification.value=Open peer info at avatar or dispute and press: {0}
 
-setting.info.headline=New XMR Autoconfirm feature
-setting.info.msg=When selling BTC for XMR you can use the autoconfirm feature to verify that the correct amount of XMR has been deposited to your wallet and automatically confirm the trade, releasing BTC to the counterparty.\n\n\
-  This can make the trade process more efficient for both peers.  Limits can also be set on the max amount of BTC that can be auto-confirmed.  See the auto-confirm section on this page for more info and to enable the feature.
+setting.info.headline=New XMR Auto confirm feature
+setting.info.msg=When selling BTC for XMR you can use the auto confirm feature to verify that the correct amount of XMR has been deposited to your wallet and automatically confirm the trade, releasing BTC to the counterparty.\n\n\
+  This can make the trade process more efficient for both peers.  Limits can also be set on the max amount of BTC that can be auto-confirmed.  See the auto-confirm settings on this page for more info and to enable the feature.\n\n\
+  Auto confirmation relies on Bisq checking the XMR transaction on an explorer API.  For maximum security it is recommended that you point Bisq to your own explorer service, but there are also services available run by Bisq (the default setting).\n\n\
+  For more information and details on how to setup, see the blog post here: [.....]
 
```

Comment:
This text assumes it will be shown on the settings screen:
```
setting.info.msg=When selling BTC for XMR you can use the auto-confirm feature to verify that the correct amount of 
XMR was sent to your wallet so that Bisq can automatically mark the trade as complete, making trades quicker for everyone.\n\n
Auto-confirm checks the XMR transaction on at least 2 XMR explorer nodes using the private transaction key provided 
by the XMR sender. By default, Bisq uses explorer nodes run by Bisq contributors, but we recommend running your
own XMR explorer node for maximum privacy and security.\n\n
You can also set the maximum amount of BTC per trade to auto-confirm as well as the number of required 
confirmations here in Settings.\n\n
See more details (including how to set up your own explorer node) on the Bisq wiki: [.....]
```
@jmacxx @chimp1984 is there a reference somewhere for how to set up an XMR explorer node?

Reply:
The service requires building first [monero](https://github.com/monero-project/monero) and then the [blockchain explorer](https://github.com/moneroexamples/onion-monero-blockchain-explorer) from source.  (Looking in the issues list, many people have had problems building due to the C++ boost library).

The only sources of information I know of are:
https://github.com/moneroexamples/onion-monero-blockchain-explorer#compilation-on-ubuntu-16041804
which further references:
https://github.com/moneroexamples/monero-compilation/blob/master/README.md

I had a terrible time trying to build on Ubuntu 16.04 (& failed), but it worked ok on 18.04 following the above instructions.

Reply:
Thanks! I've collected what I could in this article:
https://bisq.wiki/Draft:Auto-confirming_trades

It's in the draft namespace so it shouldn't be easily visible just yet.

Feel free to suggest any changes, missing details, etc (something I wondered: how is explorer address found/determined for putting into Bisq?).

Reply:
:+1:  Looks good to me.  The explorer address will be the address:port of the machine you are running it on, for example `192.168.1.111:8081` if on your LAN, or `localhost:8081` if running on the same machine as Bisq.

Reply:
@jmacxx for what it's worth you can probably include the non-draft wiki URL [in the strings](https://github.com/bisq-network/bisq/pull/4421/files#diff-2b16c2ca0e92ce28cd7268b07a143647R1253) for this PR and I'll make it live when it's merged.

https://bisq.wiki/Auto-confirming_trades

Reply:
Done, thanks!